skipper.go

```diff
@@ -783,6 +785,9 @@ type Options struct {
 	// ClusterRatelimitMaxGroupShards specifies the maximum number of group shards for the clusterRatelimit filter
 	ClusterRatelimitMaxGroupShards int
 
+	// Enable proxy protocol support
+	ProxyProtocolEnable bool
```

Comment:
Same thing on the comments goes here.

Reply:
Also I'd change the naming from `ProxyProtocolEnabled` to `EnableProxyProtocol` - the same way as other fields are named, look at `EnableRatelimiters` or `EnableRouteLIFOMetrics`.